packages/tempo/doc/tempo.ticker.md

```
@@ -85,9 +98,56 @@ setTimeout(async () => {
+ticker.on('pulse', (t) => console.log('Listener B:', t));
+```
+
+### 4. Manual Pulsing (.pulse)
+In some scenarios, you may want to drive a ticker manually (e.g., from a UI event or a WebSocket message) while still benefiting from the ticker's internal state management and listeners.
+
+```typescript
+const ticker = Tempo.ticker({ seconds: 1 }); // Still has a 1s duration logic
+// ...
+ticker.pulse(); // Manually advance and notify listeners
+```
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)